app/views/tag/_tags_popover.html.erb

```diff
@@ -3,7 +3,11 @@
 	<%= form_with remote: true, url: '/subscribe/multiple/tag/:tagnames' do |f| %>
 		<% tags.each do |tag| %>
 			<div>
-				<input type="checkbox" id="<%= tag.id %>" name="tagnames[]" value="<%= tag.name %>"><span class="label <%= label_name %>" style="margin-left: 3px; margin-right: 10px; display: inline-block;"> <%= tag.name %> </span>
+				<% if current_user.following(tag.name) %>
```

Comment:
One important question I am missing here tell me the current flow for the user. Which one of them is correct?
User is not signed in to PL. User clicks on subscription button
A. Login modal pops up .....
B. Pop up comes and the user is able to select the tags from the tag list

Reply:
Here, B is happening, but the user is redirected back to profile page and has to select tags again. I think this isn't very valid. I think I should implement A, that's more sensible.

Reply:
Absolutely

Reply:
Please change this to A.